Backend/Models/Message.cs

```diff
@@ -2,6 +2,25 @@
 {
     internal abstract class Message
     {
+        public enum MessageType
+        {
+            TextMessage,
+            CommandMessage
+        }
+
+        public static MessageType GetMessageType(string rawString)
+        {
+            string[] messageParams = rawString.Split("\r\n");
+            string doLine = messageParams[0];
+            string[] doParams = doLine.Split(" ");
+            string doArgument = doParams[1];
+            if (doArgument.Equals("SEND"))
```

Comment:
Lots of string manipulation operations here a bit expensive for a preliminary check don't you think?
If DO is present will it always be the first in the raw string?

Assuming that is that case couldn't we do something like rawstring.Substring() with the right indexes to see if SENd is present otherwise automatically go to command

Reply:
DO will always be the first line yes.
I'll give it a shot.

Reply:
Done 👍